root/defaults/authentik_location.conf

```diff
@@ -0,0 +1,24 @@
+    # all requests to /outpost.goauthentik.io must be accessible without authentication. Change proxy_pass if needed. This is the default container name.
+    location /outpost.goauthentik.io {
+        proxy_pass          http://authentik_server_1:9000/outpost.goauthentik.io;
+        # ensure the host of this vserver matches your external URL you've configured
+        # in authentik
+        proxy_set_header    Host $host;
+        proxy_set_header    X-Original-URL $scheme://$http_host$request_uri;
+        add_header          Set-Cookie $auth_cookie;
+        auth_request_set    $auth_cookie $upstream_http_set_cookie;
+
+        # required for POST requests to work
+        proxy_pass_request_body off;
+        proxy_set_header Content-Length "";
+    }
+
+    # Special location for when the /auth endpoint returns a 401,
+    # redirect to the /start URL which initiates SSO
+    location @goauthentik_proxy_signin {
```

Comment:
How/when is this used? I would have thought there would be something along the lines of
```nginx
error_page 401 =200 @goauthentik_proxy_signin;
```
being put somewhere?